src/include/sof/schedule/ll_schedule_domain.h

```diff
@@ -192,6 +203,20 @@ static inline void domain_disable(struct ll_schedule_domain *domain, int core)
 	}
 }
 
+#if CONFIG_CROSS_CORE_STREAM
+static inline void domain_block(struct ll_schedule_domain *domain)
+{
+	if (domain->ops->domain_block)
```

Comment:
ping - I think this will be difficult to quickly root cause or eliminate this as a cause for glitch without this debug logging i.e. only logged when debug log is enabled in Kconfig

Reply:
Yes, the log here would be helpful. However, this is a header file, our logging mechanism requires to pass some context parameter, not yet sure how such context var is declared when calling log from header file.

Reply:
Added logging to zephyr_domain_block() and zephyr_domain_unblock().